lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -425,17 +438,7 @@ class PinnedMessagesController extends State<PinnedMessages>
   }
 
   void _updateEventsNotifier(List<Event?> events) {
-    try {
-      eventsNotifier.value = events;
-    } on FlutterError catch (exception) {
-      Logs().e(
-        'PinnedMessages::_updateEventsNotifier():: FlutterError $exception',
-      );
-    } catch (exception) {
-      Logs().e(
-        'PinnedMessages::_updateEventsNotifier():: ErrorCode $exception',
-      );
-    }
+    eventsNotifier.value = events;
```

Comment:
why remove `try - catch`?

Reply:
because, exception only throw when we use it not correctly, its lifecycle error.

Reply:
now we ensure that we dont use the valuenotifier after dispose, so no need to catch error